_posts/2022-03-07-gsoc2022.md

```diff
@@ -0,0 +1,126 @@
+| 5       | [Context-aware guessers]({{ site.github.wiki }}/GSoC-2022-Project-Ideas#project-5-context-aware-guessers) | medium | 350 hours   | Extend how the library guesses properties such as bonds, masses or atom symbols; and write guessers that know about the context of the system (database of origin, force field...) | Python, Molecular modelling | @jbarnoud @micaela-matta @IAlibay |
+
+# Information for Students
+
+You must meet our [own requirements] if you want to be a student with MDAnalysis
+this year (read all the docs behind these links!). You must also meet the
+[eligibility criteria]. Our [GSoC FAQ][] collects common questions
+from students.
+
+The MDAnalysis community values diversity and is committed to
+providing a productive, harassment-free to every member. Our [Code of
+Conduct] explains the values that we as a community uphold. *Every
+member (and every GSoC student) agrees to follow the [Code of
+Conduct]*.
+
+As a start to get familiar with MDAnalysis and open source development you
+should follow these steps:
+
+## Complete the Quick Start Guide
+
+We have a [Quick Start Guide] explaining the basics of MDAnalysis. You
+should go through it at least once to understand how MDAnalysis is
+used. Continue reading the [User Guide] to learn more.
+
+## Introduce yourself to us
+
+Introduce yourself on the [mailing list]. Tell us what you plan to work
+on during the summer or what you have already done with MDAnalysis.
+
+## Close an issue of MDAnalysis
+
+You must have *at least one commit in the development branch of MDAnalysis* in
+order to be eligible, i.e. you must demonstrate that you have been seriously
+engaged with the MDAnalysis project.  We have a list of [easy bugs][] and
+suggested [GSOC Starter issues][GSOC Starter] to work on in our issue tracker
+on GitHub. *We only accept one [GSOC Starter issue][GSOC Starter] per
+applicant* so that everybody gets a chance. If you want to dive deeper, tackle
+some of the other issues.
+
+We also appreciate if you write more tests or update/improve our documentation.
+
+We recommend you start your application by working on an issue. It will give
+you a better understanding of MDAnalysis as a project and improve the quality
+of your application.
+
+To start developing for MDAnalysis have a look at our
+guide on [contributing to MDAnalysis][dev-guide] and write to us on
+the [mailing list][] if you have more questions about setting up a
+development environment or how to contribute.
+
+
+— Hugo @hmacdope, Micaela @micaela-matta, Jonathan @jbarnoud, Richard @richardjgowers, Lily @lilyminium, Fiona @fiona-naughton, Irfan @IAlibay, Oliver @orbeckst
+
+
+[org]: https://summerofcode.withgoogle.com/organizations/
+[Apply]: https://summerofcode.withgoogle.com/student-signup/
```

Comment:
404 at the moment

Reply:
Unless one of us is going to remember to change this on April 4th, should we just not have the link?

Reply:
I am removing the deep link and just put the raw site URL into the text.